content/courses/connecting-to-offchain-data/verifiable-randomness-functions.md

```diff
@@ -99,21 +104,21 @@ implementation of VRF to Solana.
 Switchboard is a decentralized Oracle network that offers VRFs on Solana.
 Oracles are services that provide external data to a blockchain, allowing them
 to interact with and respond to real-world events. The Switchboard network is
-made up of many different individual oracles run by third parties to provide
-external data and service requests onchain. To learn more about Switchboard’s
-Oracle network, please refer to our
+composed of many individual oracles run by third parties, which provide external
+data and service requests onchain. To learn more about Switchboard’s Oracle
```

Comment:
This is an error in the old material, but 'oracle' is a concept and should be lowercase (I think people may be confused with the database company). Fix this everywhere please!